src/views/transaction/transaction.view.jsx

```diff
@@ -60,7 +60,7 @@ function Transaction ({
   const classes = useTransactionStyles()
   const { search } = useLocation()
   const urlSearchParams = new URLSearchParams(search)
-  const accountIndex = urlSearchParams.get('accountIndex')
+  let accountIndex = urlSearchParams.get('accountIndex')
```

Comment:
maybe use useState?

Reply:
Yep, I would use `useState` instead of variable reassignment here

Reply:
Fixed in a different way.